### DISCUSSION TITLE
Does sorting by popularity still work as expected?

### DISCUSSION THREAD

**Discussion by s-h-a-d-o-w** [community/community#140448] [npm]
I'm pretty sure that ever since I can remember, popularity equaled weekly downloads? Yet, looking at this: https://www.npmjs.com/search?ranking=popularity&q=css%20reset

... the results are:
1. 6,072,580
2. 347,873
3. 195,892
4. 575,247
5. 88,947
6. 339,281

(Although I don't rule out that it's always been like this and I just never noticed for some reason, even though I always check weekly downloads during decision making.)

**Comment by s-h-a-d-o-w** [comment]
Now we can sort by downloads but... these don't look like css reset libraries:

![npm_css_reset](https://github.com/user-attachments/assets/ddc584c7-715c-48e9-aaa7-f238673a93f1)

(Yes, this bug has been around for weeks, possibly months. But I just randomly stumbled across this thread again.)

**Reply by kpachbiu88** [reply]
It seems to search work by keywords in readme, not by meaning

**Reply by s-h-a-d-o-w** [reply]
Yes. But I'm pretty sure that it didn't used to be like that and it's virtually unusable now because of this. Especially since one can't use e.g. "AND" to make it search for both keywords.

I've started using npms.io (which unfortunately doesn't seem to get updated any more) and perplexity to search for libraries instead. I never had to in the past. The npmjs search was always fine (at least for initial research) up until those few months ago.
